### ISSUE TITLE
Add `proposal-export-namespace-from` to `preset-env`

### ISSUE THREAD

**Issue by billyjanitsch** [babel/babel#11363]
## Bug Report

- [x] I would like to work on a fix!

**Current Behavior**
When `@babel/preset-env` is enabled, export namespace syntax causes Babel to emit an error:

```
SyntaxError: /[...]/index.js: Unexpected export specifier type
> 1 | export * as ns from './foo'
```

A different, less descriptive error is emitted when `{loose: true}` is passed to the preset:

```
TypeError: /[...]/index.js: Cannot read property 'name' of undefined
```

**Input Code**
See [this repo](https://github.com/billyjanitsch/babel-export-ns-bug) for a minimal reproduction. Run `npm i && npm run build` to observe the error. Removing `@babel/preset-env` from the config prevents the error. So does providing `{modules: false}` to the preset.

```js
export * as ns from './foo'
```

**Expected behavior/code**
I would expect this code to transpile without error.

**Babel Configuration (babel.config.js, .babelrc, package.json#babel, cli command, .eslintrc)**
- Filename: `babel.config.js`

```js
module.exports = {
  // comment out this line to fix the build:
  presets: ['@babel/env']
}
```

**Environment**

```
  System:
    OS: macOS Mojave 10.14.6
  Binaries:
    Node: 13.11.0 - ~/.config/nvm/13.11.0/bin/node
    npm: 6.14.4 - ~/.config/nvm/13.11.0/bin/npm
  npmPackages:
    @babel/cli: ^7.8.4 => 7.8.4 
    @babel/core: ^7.9.0 => 7.9.0 
    @babel/preset-env: ^7.9.0 => 7.9.0
```

**Comment by nicolo-ribaudo** [comment]
@billyjanitsch The problem is that we added `export * as ns from "foo"` support by default to `@babel/parser` (in #10521) but not to `@babel/preset-env`.

As a workaround, you can enable `@babel/plugin-proposal-export-namespace-from` in your config. A proper fix would be similar to https://github.com/babel/babel/commit/80aa7dc3c6972c0b107dd8b4ed88663141cf8583#diff-cf817afef443d0d96acf65275e0edf13.

---



If it is the first time that you contribute to Babel, follow these steps: (you need to have `make` and `yarn` available on your machine)
1. **Write a comment there** to let other possible contributors know that you are working on this bug.
1. Fork the repo
1. Run `git clone https://github.com/<YOUR_USERNAME>/babel.git && cd babel`
1. Run `yarn && make bootstrap`
1. Wait :hourglass_flowing_sand: 
1. Run `make watch` (or `make build` whenever you change a file)
1. Add a test (only `input.js`; `output.js` will be automatically generated)
1. Update the code!
1. `yarn jest preset-env` to run the tests
    - If some test outputs don't match but the new results are correct, you can delete the bad `output.js` files and run the tests again
    - If you prefer, you can run `OVERWRITE=true yarn jest preset-env` and they will be automatically updated.
1. If it is working, run `make test` to run all the tests
1. Run `git push` and open a PR!

**Comment by JLHwung** [comment]
@nicolo-ribaudo Actually it will be more than 80aa7dc because `export-ns-from` is not available in either `compat-table` or `caniuse-db`. We may resort to unconditionally including `proposal-export-namespace-from` due to lack of support data.

**Comment by JounQin** [comment]
Any news on this? Is there anything we can do to resovle this issue?

**Comment by amoshydra** [comment]
@JounQin 

> As a workaround, you can enable @babel/plugin-proposal-export-namespace-from in your config
> -- https://github.com/babel/babel/issues/11363#issuecomment-607830293

See: https://babeljs.io/docs/en/babel-plugin-proposal-export-namespace-from

**Comment by JLHwung** [comment]
This issue has been fixed by #11849
